### ISSUE TITLE
Database configuration using port isn't working

### ISSUE THREAD

**Issue by brain64bit** [madebyhiro/codem-transcode#39]
When using PostgreSQL as database i need to specified the **port** because postgres running on port **5432** so my configuration would be look like :

```
{
    "port":            8080,
    "access_log":      "/var/log/access.log",
    "database":{
      "dialect": "postgres",
      "username": "user",
      "password": "pass",
      "database": "codem_db",
      "host": "localhost",
      "port": 5432
    },
    "slots":           8,
    "interface":       "127.0.0.1",
    "encoder":         "ffmpeg",
    "scratch_dir":     "/tmp",
    "use_scratch_dir": true,
    "ffprobe":         null
}
```
but unfortunately i've got exception like this :
```
13 Feb 00:47:05 - Started server on interface 127.0.0.1 port 8080 with pid 1577.
13 Feb 00:47:05 - Caught exception: Error: connect ECONNREFUSED
13 Feb 00:47:05 - Caught exception: Error: Connection terminated
```

After that i'm trying to debug on **connect(port, host)** function inside **../node_modules/pg/lib/connection.js**, i've found wrong value on port argument is set to **3306**. And then it seem inside **../node_modules/codem-transcode/lib/job.js** on **JobUtils.getDatabase** function you're not set the port :
```
getDatabase: function() {
    if (JobUtils.sql == null) {
      if (config['database']['dialect'] == "sqlite") {
        JobUtils.sql = new Sequelize('database', 'username', 'password', {
          dialect: 'sqlite',
          storage: config['database']['database'],
          logging: false
        });
      } else {
        JobUtils.sql = new Sequelize(config['database']['database'], config['database']['username'], config['database']['password'], {
          dialect: config['database']['dialect'],
          storage: config['database']['database'],
          host: config['database']['host'],
          logging: false
        });
      }
    }
    return JobUtils.sql;
  }
```

**Comment by brain64bit** [comment]
My workaround for this is :
```
getDatabase: function() {
    if (JobUtils.sql == null) {
      if (config['database']['dialect'] == "sqlite") {
        JobUtils.sql = new Sequelize('database', 'username', 'password', {
          dialect: 'sqlite',
          storage: config['database']['database'],
          logging: false
        });
      } else {
        JobUtils.sql = new Sequelize(config['database']['database'], config['database']['username'], config['database']['password'], {
          dialect: config['database']['dialect'],
          storage: config['database']['database'],
          host: config['database']['host'],
          port: config['database']['port'],
          logging: false
        });
      }
    }
    return JobUtils.sql;
  },
```
and its work for me.

**Comment by tieleman** [comment]
Fixed in 0.5.7.
